rosdep/base.yaml

```diff
@@ -310,6 +310,27 @@ boost:
     xenial: [libboost-all-dev]
     yakkety: [libboost-all-dev]
     zesty: [libboost-all-dev]
+boost-program-options:
+  ubuntu:
```

Comment:
Please fill at least debian rules as well. And Fedora and OSX would be great to have as well. And link to the package listings online for easier review. 

https://github.com/ros/rosdistro/blob/master/CONTRIBUTING.md

Reply:
I didn't find it for debian wheezy, but added fedora and debian (jessie + stretch).